lib/shared/sdk/standard/index.js

```diff
@@ -71,15 +71,21 @@ class StandardAdapter extends EventEmitter {
       if (_.includes([ '0001', 'RPi-MSD- 0001', 'File-Stor Gadget', 'Linux File-Stor Gadget USB Device' ], drive.description)) {
         drive.description = 'Compute Module'
         drive.icon = 'raspberrypi'
-        drive.system = false
+        drive.isSystem = false
       }
 
       return drive
     }).catch((error) => {
       this.emit('error', error)
       callback && callback(error)
     }).filter((drive) => {
-      return options.includeSystemDrives || !drive.system
+      return !drive.error && (options.includeSystemDrives || !drive.isSystem)
+    }).map((drive) => {
+      drive.displayName = drive.device
```

Comment:
Can we move the displayName to `drivelist`, given we're likely going to use the same thing on the CLI? Of course doesn't need to be done right now

Reply:
I was thinking that this logic is very application specific – in general the `description` is the best display name, imo, so the CLI might just want to switch to that?
I just recreated the displayName here, as we've got this fairly specific difference in handling them on Win in contrast to Linux / Mac in Etcher.
What do you think? Of course we can add it back to drivelist also, but I feel that it shouldn't be concerned with these UI specific things.